README.md

```diff
@@ -213,6 +213,38 @@ this.addinClientService.addEventHandler({
 });
 ```
 
+You can send a custom event to the host page, as long as the host supports the event type.
+Before sending an event to the host page, you must subscribe to the AddinClientService's init `args`
+event and check the `supportedEventTypes` property to determine what event types the host page
+will handle.
+Sending custom events to the host page is currently only supported for Blackbaud internal use.
+
+```js
+this.addinClientService.args.subscribe((args: AddinClientInitArgs) => {
+  this.supportedEventTypes = args.supportedEventTypes;
+
+  args.ready({
+    showUI: true
+  });
+});
+
+// Before sending event, check to make sure the event is supported by the host page
+if (this.supportedEventTypes.includes('my-event-type')) {
+  // To send add-in events to the host, call the sendEvent method
+  this.addinClientService.sendEvent({
+    type: 'my-event-type',
+    context: { /* arbitrary context object to pass to host page */ }
```

Comment:
same feedback I gave on the other PR here for "arbitrary"

Reply:
Fixed by removing "arbitrary"